views/index.erb

```diff
@@ -96,7 +96,7 @@
   <div class="row-fluid">
     <div class="span9">
       <strong>Shopper Email:</strong>&nbsp;&nbsp;
-      <input class="input-large" type="text" id="shopperEmail" value="<%= ENV['DEFAULT_SHOPPER_EMAIL']||'carduser@example.com'%>">
+      <input class="input-large" type="text" id="shopperEmail", placeholder="user@example.com">
```

Comment:
Let's just remove this whole section, and allow the user to enter their email address in the modal like they do everyone else.